packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,155 @@
+# Nimble Accordion
+
+## Overview
+
+The `nimble-accordion` is a vertical stack of interactive headings. The headings are controls that can be used to reveal or hide content in specific sections. The Nimble accordion is based upon [FAST's accordion component](https://github.com/microsoft/fast/tree/archives/fast-element-1/packages/web-components/fast-foundation/src/accordion).
+
+### Background
+
+[Nimble issue #533: nimble-accordion Component](https://github.com/ni/nimble/issues/533)
+
+[Visual Design](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-85131&mode=design&t=DxDRlqT7MeCPLOxi-0)
+
+There are several open questions with the current visual design. They're tracked in this GitHub issue [#1399](https://github.com/ni/nimble/issues/1399).
```

Comment:
The issue was closed before this PR went in, so this line can be removed and anything that needs to be captured in the spec should be updated